geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/LauncherLifecycleCommands.java

```diff
@@ -771,11 +771,18 @@ public Result statusLocator(
               CliStrings.STATUS_SERVICE__GFSH_NOT_CONNECTED_ERROR_MESSAGE, LOCATOR_TERM_NAME));
         }
       } else {
-        final LocatorLauncher locatorLauncher =
-            new LocatorLauncher.Builder().setCommand(LocatorLauncher.Command.STATUS)
-                .setBindAddress(locatorHost).setDebug(isDebugging()).setPid(pid)
-                .setPort(locatorPort).setWorkingDirectory(workingDirectory).build();
+        final LocatorLauncher locatorLauncher;
 
+        if ((locatorHost == null) && (locatorPort == null) && (workingDirectory == null)) {
```

Comment:
What if `locatorPort` is specified, but `locatorHost` is not?  It seems like that might pass this check, but still result in an error.